src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -1042,5 +1042,5 @@ messages.imsg.m0087 = Not permitted to merge account
 
 frmelmnts.lbl.otpValidationFailed = OTP validation failed.
 frmelmnts.lbl.generateOtpFailed = Generate OTP failed. Please try again
-
-frmelmnts.lbl.certificates = Certificates
\ No newline at end of file
+messages.stmsg.m0144 = You do not have appropriate Rights to access this page.
```

Comment:
Fix the Rights spelling. Make it lowercase

Reply:
Done